tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -2723,6 +2723,26 @@ def take_while(self, predicate):
 
     return _TakeWhileDataset(self, predicate)
 
+  def unique(self):
+    """A transformation that discards duplicate elements of a `Dataset`.
+
+    Use this transformation to produce a dataset that contains one instance of
+    each unique element in the input. For example:
+
+    >>> dataset = tf.data.Dataset.from_tensor_slices([1, 37, 2, 37, 2, 1])
+    >>> dataset = dataset.unique()
+    >>> sorted(list(dataset.as_numpy_iterator()))
+    [1, 2, 37]
+
+    NOTE: This transformation is only applicable on datasets with either
+      `tf.int32`, `tf.int64` or `tf.string` type elements.
```

Comment:
The documentation should also mention that the transformation does not support datasets that do not fit into memory.

Reply:
modified the `NOTE` to convey this information.